### ISSUE TITLE
add voting and comments to photos

### ISSUE THREAD

**Issue by nicHoch** [theCrag/website#1062]

**Comment by lordyavin** [comment]
Thumbs up if it would be possible to comment on photos!

**Comment by scd** [comment]
Yes eventually. I am hoping comments on photos will be resolved for the streams work I am currently undertaking.

**Comment by Drazhar** [comment]
Commenting seems to be working, but what about the ability to like photos?
I would give this a :+1:
